src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import Event.Event;
+import Event.Deadline;
+import Event.ToDo;
+import Event.Task;
+import RuntimeSupport.DukeException;
+import java.util.Scanner;
+
 public class Duke {
+    static String BREAK_LINE = "____________________________________________________________";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println(BREAK_LINE + "\nHello! I'm 550W.\nWhat can I do for you?\n" + BREAK_LINE);
+        Task[] tasks = new Task[100];
+        int count = 0;
+        Scanner in = new Scanner(System.in);
+        String line = "";
+
+        while (true) {
+            try {
+                line = in.nextLine();
+                System.out.println(BREAK_LINE);
+
+                if (line.equals("bye")) {
+                    break; //Break out of the loop immediately when bye command is entered, quitting the programme.
+                }
+
+                if (line.equals("list")) {
+
+                    System.out.println("Abracadabra! \uD83C\uDF1F Here are the tasks in your list:");
+                    for (int i = 0; i < count; i++) {
```

Comment:
You could consider refactoring to avoid deep nesting this code as well. ([guideline](https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-deep-nesting))